server/config/db-connect.js

```diff
@@ -0,0 +1,30 @@
+//db-connect.js
+/* Created by Fil on August 25, 2019 */
+
+//TODO (FG) - Change require to import once Babel works?
```

Comment:
does babel not work if you run npm run dev?

Reply:
No I am getting an error on running the server.  My assumption was that it was Babel but haven't really dug in to it. The client starts up fine.